MusicTaste/MusicTaste/MatchingAlgorithm.h

```diff
@@ -0,0 +1,17 @@
+#import <Foundation/Foundation.h>
+#import <Parse/Parse.h>
+#import <CoreData/CoreData.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@interface MatchingAlgorithm : NSObject
+
+@property (strong, nonatomic) NSArray *foundPosts;
++ (instancetype)shared;
++ (void) lookForMatches;
++ (void) compare:(PFObject *)potentialMatch withDictionary:(NSMutableDictionary *)matches withData:(NSManagedObject *)data;
++ (double) compareSpotifyData:(PFObject *)potentialMatch withData:(NSManagedObject *)userSpotify;
```

Comment:
Create the three functions used for the Matching Algorithm: look for matches, compare, and compare spotifyData